scripts/ElaboratorListener_h.py

```diff
@@ -0,0 +1,203 @@
+import config
+import file_utils
+
+
+def _generate_module_listeners(models):
+    listeners = []
+    classname = 'module'
+    while classname:
+        model = models[classname]
+
+        for key, value in model.allitems():
+            if key in ['class', 'obj_ref', 'class_ref', 'group_ref']:
+                name = value.get('name')
+                type = value.get('type')
+                card = value.get('card')
+
+                cast = 'any' if key == 'group_ref' else type
+                Cast = cast[:1].upper() + cast[1:]
+                method = name[:1].upper() + name[1:]
+
```

Comment:
Nicely readable section of what happens for different cardinalities and method types!

Reply:
I tried to keep it modular rather than cram everything into a single large script. Hopefully, it would be easier for non-python developers to follow.